charts/plugin-site/values.yaml

```diff
@@ -2,12 +2,46 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
 
-image:
-  repository: jenkinsciinfra/plugin-site
-  tag: 122-bd2a5f
-  pullPolicy: IfNotPresent
+backend:
+  replicaCount: 1
+  image:
+    repository: jenkinsciinfra/plugin-site-api
+    tag: 133-b6e652
+    pullPolicy: IfNotPresent
+  port: 8080
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
+    requests:
+      cpu: 500m
+      memory: 1024Mi
+
+frontend:
+  replicaCount: 1
+  image:
+    repository: jenkinsciinfra/plugin-site
+    tag: 122-bd2a5f
+    oldMode: true
+    #repository: halkeye/jenkins-plugin-site
+    #tag: 44-14cfe6
+    pullPolicy: IfNotPresent
+  port: 80
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
+    requests:
+      cpu: 1000m
+      memory: 1024Mi
```

Comment:
The frontend doesn't probably need that amount of memory and cpu.

Reply:
the javascript version does, I had it set locally to 100m cpu, and it took 15 min to test, compile, and startup.
at 1000m, it took a couple seconds

The old image is really not optimized in any way.

Reply:
Just need to remember to update the resources when we enable the new static site